caffe-grid/src/main/scala/com/yahoo/ml/caffe/Logging.scala

```diff
@@ -0,0 +1,157 @@
+// Copyright 2016 Yahoo Inc.
```

Comment:
From the comments within the code below:

    Copied from org.apache.spark.internal for use within CaffeOnSpark

The only other way is to write a new class that has the same functionality. Why do that?  If you feel strongly about then we can put a TODO.